include/modem/nrf_modem_lib_trace.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**@file nrf_modem_lib_trace.h
+ *
+ * @defgroup nrf_modem_lib_trace nRF91 Modem trace module
+ * @{
+ */
+#ifndef NRF_MODEM_LIB_TRACE_H__
+#define NRF_MODEM_LIB_TRACE_H__
+
+#include <stdint.h>
+
+/** @brief Initialize the modem trace module.
+ *
+ * Initializes the module and the trace transport medium.
+ *
+ * @return Zero on success, non-zero otherwise.
+ */
+int nrf_modem_lib_trace_init(void);
```

Comment:
Why are you prefixing with `nrf_modem_lib_`? Shouldn't this just be `nrf_modem_`?

Reply:
I believe you couldn't see it because it was a resolved conversation. I have unresolved it for you to have a look. Basically the reason is to distinguish this API from the APIs implemented by libmodem (in `nrf_modem.h` in sdk-nrfxlib repo). Correct me if I am wrong @lemrey

Reply:
OK, I can't say I am happy about this but it will have to do. Thanks for the patience!